spec/lib/fields/belongs_to_spec.rb

```diff
@@ -66,4 +66,19 @@
       end
     end
   end
+
+  describe "#resources" do
+    context "with `order` option" do
+      it "returns the resources in correct order" do
+        5.times { FactoryGirl.create(:customer) }
+        options = { order: "name" }
+        association = Administrate::Field::BelongsTo.with_options(options)
+        field = association.new(:customers, [], :view)
+
+        correct_order = Customer.all.order("name").map(&:id)
+        resources = field.associated_resource_options.drop(1).map { |r| r[1] }
```

Comment:
I wonder if this line is a little complex. Do you think we'd be able to simplify it a little?